src/main/resources/alfresco/ibatis/org.alfresco.repo.domain.dialect.Dialect/solr-common-SqlMap.xml

```diff
@@ -115,12 +116,13 @@
     <select id="select_Txn_Nodes" parameterType="SOLRNodeParameters" resultMap="result_Node">
         select
             COALESCE(np.long_value, node.id) as id,
-            node.type_qname_id  as type_qname_id,
-            node.transaction_id as txn_id,
-            node.uuid           as uuid,
-            node.acl_id         as acl_id,
-            store.protocol      as protocol,
-            store.identifier    as identifier,
+            node.type_qname_id      as type_qname_id,
+            node.transaction_id     as txn_id,
+            txn.commit_time_ms      as txn_commit_time_ms,
+            node.uuid               as uuid,
+            node.acl_id             as acl_id,
+            store.protocol          as protocol,
+            store.identifier        as identifier,
```

Comment:
This change includes only "txn.commit_time_ms" in the query, but it looks more complex due to re-indentation... Probably all these changes can be adjusted to reflect only the real change.